lib/newPlotTimesFreqfig.m

```diff
@@ -0,0 +1,68 @@
+function [] = newPlotTimesFreqfig(t,f,T,F,friv)
+
+%   plot of signal in time in the intTime selected and their fft, we are
+%   passing matrices containing temporal (T) and frequency (F) values with 
+%   their corresponding values of signals both in time domain and FFT; if
+%   'friv' is given, then we have a plot showing normalized frequency, if 0
+%   values are not normalized.
```

Comment:
please add a comment about `t` and `f` variables, saying that these are the arrays for the horizontal axes, and if they have only one column, then it means that all the time signals and FFT have the same domain and resolution.